statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1309,7 +1309,7 @@ def resid_pearson(self):
     @cache_readonly
     def resid_working(self):
         # Isn't self.resid_response is already adjusted by _n_trials?
-        val = (self.resid_response / self.family.link.deriv(self.mu))
+        val = (self.resid_response * self.family.link.deriv(self.mu))
         val *= self._n_trials
```

Comment:
self.mu in binomial is in terms of fraction or probability. 
resid_response is in counts (taking n_trials into account)
So, I guess the multiplication by `_n_trials` is to adjust the `self.family.link.deriv(self.mu))`